static/templates/bottom_whitespace.hbs

```diff
@@ -0,0 +1,12 @@
+<div id="next_unread_container">
+    <a type="button" class="button animated-purple-button next_topic tippy-zulip-tooltip" data-tippy-content="Narrow to an unread topic (n)" data-tippy-placement="bottom">Next unread topic</a>
+    <a type="button" class="button animated-purple-button next_pm tippy-zulip-tooltip" data-tippy-content="Narrow to an unread PM (p)" data-tippy-placement="bottom">Next unread PM</a>
+    <a type="button" href="#recent_topics" class="button animated-purple-button">Recent topics</a>
+</div>
```

Comment:
Should these have the `animated-purple-button` class?  

Also, should the "Recent topics" button have a tooltip with the `(t)` keyboard shortcut indicated?

Reply:
> Should these have the animated-purple-button class?

Yes, we just change the background color for the button here, rest of the styles are borrowed from it.

> Also, should the "Recent topics" button have a tooltip with the (t) keyboard shortcut indicated?

I wasn't sure of the tooltip text, added it anyway if you want to edit.